app/components/Form/ProjectContactFormSummary.tsx

```diff
@@ -185,7 +191,9 @@ const ProjectContactFormSummary: React.FC<Props> = ({
               tagName={"dl"}
               theme={readOnlyTheme}
               fields={renderDiff ? customFields : fields}
-              schema={projectContactSchema as JSONSchema7}
+              schema={
+                primaryContact?.node.formByJsonSchemaName.jsonSchema.schema
+              }
```

Comment:
cast to `JSONSchema7`?

Reply:
I think the db schemas are already type `JSONSchema7`